File: app/src/main/java/com/example/sportapp/support2026/features/news/data/dto/newsList/components/NewsDto.kt
package com.example.sportapp.support2026.features.news.data.dto.newsList.components

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
class NewsDto (
    @SerialName("id") val id: Int?,
    @SerialName("dateTime") val dateTime: String?,
    @SerialName("sport") val sport: String?,
    @SerialName("imageId") val imageId: String?,
    @SerialName("articleText") val text: String?,
    @SerialName("title") val title: String?,
)

File: SportAppServer/SportAppServer/SportAppServer/Support2026/Application/Feature/NewsFeature/Dto/NewsDto.cs
using Newtonsoft.Json;
using System.Text.Json.Serialization;

namespace SportAppServer.Support2026.Application.Feature.NewsFeature.Dto
{
    public class NewsApiDto
    {
        [JsonProperty("id")]
        public int Id { get; set; } 

        [JsonProperty("date_time")]
        public DateTime DateTime { get; set; }

        [JsonProperty("sport")]
        public string Sport { get; set; } = "";

        [JsonProperty("title")]
        public string Title { get; set; } = "";

        [JsonProperty("image_id")]
        public string ImageId { get; set; } = "";

        [JsonProperty("text")]
        public string ArticleText { get; set; } = "";
    }
}
